### PR TITLE
Updating CODEOWNERS for Communication Identity & Common Packages

### PR DESCRIPTION
**Pull request by mjafferi-msft**
# Description
Updating CODEOWNERS for Communication Identity & Common Packages

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: .github/CODEOWNERS
Comment (AikoBB) [inline .github/CODEOWNERS:88]: I think we provided here the wrong path, should it be as follow?
`/sdk/communication/**/**/**/**/_shared/`

Comment (AikoBB) [reply .github/CODEOWNERS:88]: also, do we have to add here the shared folder under the `test` folder as well?

Comment (mjafferi-msft) [reply .github/CODEOWNERS:88]: Yes, the path for the shared folder should be : /sdk/communication/**/**/**/**/_shared/

Comment (AikoBB) [reply .github/CODEOWNERS:88]: let's wait for other's response for the shared folder within the test folder 
cc: @petrsvihlik @maximrytych-ms @ostoliarova-msft

**Review by jsquire — APPROVED** [review]
Absent the discussion on paths, the baseline change looks good.

File: .github/CODEOWNERS
Comment (petrsvihlik) [reply .github/CODEOWNERS:88]: I was under the impression that `**/_shared/` matches the `_shared` folder at any depth....

https://stackoverflow.com/a/67224003/1332034
image

@jsquire do you know if that's the case?

**Review by petrsvihlik — APPROVED** [review]

**Comment by jsquire** [comment]
> I was under the impression that **/_shared/ matches the _shared folder at any depth....

@petrsvihlik : I wouldn't consider me the authoritative source on this, but that is my understanding as well.   The normal splat pattern semantics use the double-star to indicate "anything below this root."

File: .github/CODEOWNERS
Comment (petrsvihlik) [reply .github/CODEOWNERS:88]: @mjafferi-msft could you please create a testing (draft) PR with a change just in the _shared folder to see if the right people are included? once verified, we'll close that PR without merging.

Comment (petrsvihlik) [reply .github/CODEOWNERS:88]: works correctly as it is now - #27665

**Pull request merged**
